compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/DesktopPlatform.desktop.kt

```diff
@@ -35,5 +36,20 @@ internal enum class DesktopPlatform {
                 else -> Unknown
             }
         }
+
+        /**
+         * Identify OS on which the application is currently running.
+         */
+        val Current get() = overriddenCurrent ?: _current
+
+        @VisibleForTesting
+        fun withOverriddenCurrent(newCurrent: DesktopPlatform, body: () -> Unit) {
```

Comment:
Not crucial, but such functions are more useful if they're `inline`, and return what `body` returns.

Reply:
Done